scripts/build.sh

```diff
@@ -8,6 +8,20 @@ else
     ZIG_OS="macos"
 fi
 
+target_set="false"
+case "$*" in
+    *-Dtarget*)
+	target_set="true"
+	;;
+esac
+
+if [ "$target_set" = "false" ]; then
+    if [ "$(./zig/zig targets | grep '"os": "' | cut -d ":" -f 2 | cut -d '"' -f 2)" = "macos" ]; then
+	# Add target to args
+	set -- "$@" "-Dtarget=native-macos"
+    fi
+fi
```

Comment:
Moving the target check into build.sh makes more sense over requiring it externally. Could the target flag also be removed in `benchmark.sh` now?

Reply:
Ok, I gave that a shot too!